docs/reference/index.rst

```diff
@@ -8,3 +8,4 @@ Reference
    kernel-boot-parameters
    kernel-config-options
    releases
+   rt-conf-yaml
```

Comment:
We probably want a sub-menu here. What do you think @AnneCYH? There will be other rt-conf docs in the future.